Amplify/Categories/DataStore/Model/Temporal/Temporal.swift

```diff
@@ -7,23 +7,17 @@
 
 import Foundation
 
-public struct Temporal {
+/// `Temporal` is namespace to all temporal types and basic date operations. It can
+/// not be directly instantiated.
+///
+///  Available `Temporal` `Specs`
+/// - Temporal.Date
+/// - Temporal.DateTime
+/// - Temporal.Time
+public enum Temporal {}
 
-    /// This struct is used as a namespace to all temporal types. It should
-    /// not be directly instantiated.
-    ///
-    /// - seealso: Temporal.Date
-    /// - seealso: Temporal.DateTime
-    /// - seealso: Temporal.Time
-    /// - seealso: Temporal.TimeStamp
-    private init() {}
-}
-
-/// The `TemporalSpec` protocol defines an [ISO-8601](https://www.iso.org/iso-8601-date-and-time-format.html)
-/// formatted Date value. Types that conform to this protocol are responsible for providing
-/// the parsing and formatting logic with the correct granularity.
 public protocol TemporalSpec {
-
+    associatedtype Format: TemporalSpecValidFormatRepresentable
```

Comment:
For some reason, I do not see the implementation of this protocol `TemporalSpecValidFormatRepresentable` in this PR. Will post the comments I have so far, will review it after we figure out if we are missing some files in the PR

Reply:
It's in `TemporalFormat.swift`